README.md

```diff
@@ -8,6 +8,11 @@ The Spring Cloud Stream microservice generated using this template will be an _a
 
 Note that this template ignores the 'Servers' section of AsyncAPI documents. The main reason for this is because SCSt does not directly work with messaging protocols. Protocols are implementation details specific to binders, and SCSt applications need not know or care which protocol is being used.
 
+## Technical requirements
+
+- 1.8.6 =< [Generator](https://github.com/asyncapi/generator/)
```

Comment:
if this is hard requirement then you need to also update it here https://github.com/asyncapi/java-spring-cloud-stream-template/blob/master/package.json#L68 otherwise just specify that it is a requirement only in a specific case

Reply:
Oh, right. I thought that was already set but I was looking at devDependencies. I fixed it.